### DISCUSSION TITLE
Please include your own version of ARM Translation

### DISCUSSION THREAD

**Discussion by Zero3K** [Genymobile/feedback#28] [Genymotion Desktop] [answered]
The ones that I have found online do not let certain apps (such as MyZone, Dissidia: Opera Omnia) either install or they crash when ran.

**Comment by Zero3K** [comment]
Well?

**Reply by jmaneyrol69** [reply]
@Zero3K having our own ARM translation tools is on our mind but we do not have any ETA as of now.
Do not hesitate to visit and subscribe to our blog at https://www.genymotion.com/blog/ to get the latest news.
Best regards.

**Comment by agavignet** [comment, marked as answer]
That's obviously a wish from our side but as you said, we need to create our own (for legal reasons) and it's not a small tasks. If you need Native ARM Android you can either user Genymotion Desktop on Apple Silicon hardware or use Genymotion Device Image.
